package.json

```diff
@@ -19,6 +19,7 @@
   ],
   "atomShellVersion": "0.21.0",
   "dependencies": {
+    "6to5": "3.0.14",
```

Comment:
`6to5-core` would be better here since it's a build of `6to5` without certain dependencies so it'll install faster. You can see the script that generates the `6to5-core` `package.json` [here](https://github.com/6to5/6to5/blob/master/tools/generate-core-package-json).